script.js

```diff
@@ -0,0 +1,79 @@
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    } return books;
+  }
```

Comment:
- [ ] Great job on adding new book 👏 . but you have a problem with updating your local storage because it does not update with the new information and shows the same book every time you read the page :point_down: 

![1111](https://user-images.githubusercontent.com/57562869/148076793-7d167f29-96eb-42a5-8597-996313b5eaff.png)

Reply:
I do not quite understand the issue. Can you please clarify more?